packages/core/src/constants.ts

```diff
@@ -20,3 +20,5 @@ export const MAX_PARALLEL_PINGS = 14
 
 export const CHECK_TIMEOUT = 60000
 export const ACKNOWLEDGEMENT_TIMEOUT = 2000
+
+export const PEER_METADATA_PROTOCOL_VERSION = 'protocol_version'
```

Comment:
Should it be this string?

Reply:
It is true that it originates from the heartbeat sub-protocol, but the meaning is that is should refer to the overall HOPR protocol version, so I would keep it with this name.